help/en/package/jmri/jmrit/permission/PermissionManager.shtml

```diff
@@ -18,7 +18,18 @@
       <p><strong>Permissions</strong> is used to protect sensitive parts of JMRI, for example
       preventing the loading and storing of the <em>table content and panels</em> file by
       unauthorized users.</p>
-
+      
+      <ul>
+		  <li><a href="#settings">Settings</a></li>
+		  <li><a href="#defaultusers">Default users</a></li>
+		  <li><a href="#defaultpermissions">Default permissions</a></li>
+		  <li><a href="#lockedout">I have locked myself out! What do I do?</a></li>
+		  <li><a href="#securingpreferences">Securing the preferences</a></li>
+		  <li><a href="#connectionfailure">Connection failure during startup of JMRI</a></li>
+		  <li><a href="#panelproshutdown">Permission failure during PanelPro shutdown</a></li>
+      </ul>
+
+      <a id="settings"></a>
```

Comment:
The link ids don't need to be in separate "a" tags.  They can be anywhere, such as the "h2" tags.

There is no need to change.

Reply:
Thanks. I have updated the PR. Have I got it right now?